server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/VacationIntegrationTest.java

```diff
@@ -211,6 +211,9 @@ public void jmapVacationShouldSendNotificationTwiceWhenVacationReset() throws Ex
         // User 2 matthieu@mydomain.tld sends User 1 a mail
         String user2OutboxId = getOutboxId(user2AccessToken);
         sendMail(user2AccessToken, user2OutboxId, "user|inbox|1");
+        // Wait user 1 to receive the eMail before reset of vacation
+        calmlyAwait.atMost(10, TimeUnit.SECONDS)
```

Comment:
@mbaechler the default value used everywhere is 10 secondes. Why should it be different here ?

Ps : VacationIntegrationTest and SetMessagesMethodTest

+ I don't want to wait 10 minutes if my test fails